packages/core/src/styles/themes/dynamic-themes.stories.mdx

```diff
@@ -25,17 +25,12 @@ if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
 
 // automatically set high contrast styles based on OS (Edge/Firefox only) https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-contrast
 if (window.matchMedia('(-ms-high-contrast: active)')).matches || window.matchMedia('(prefers-contrast: high)')).matches) {
```

Comment:
Should we add any of the media queries outlined on this page from Microsoft? https://blogs.windows.com/msedgedev/2020/09/17/styling-for-windows-high-contrast-with-new-standards-for-forced-colors/

Reply:
I left off the high contrast light/dark queries since the tokens automatically support both using the system color values, but I'll add that link in the comment there for reference

Reply:
ah wait sorry I see its the old style not the updated queries. I'll get that fixed up!

Reply:
Added  the new style with the older version fallback. Also added a comment to the edge blog post